src/test/goldens/paper-behaviors/paper-button-behavior.d.ts

```diff
@@ -1,3 +1,10 @@
+/**
+ * DO NOT EDIT
+ *
+ * This file was automatically generated by
+ * https://github.com/Polymer/gen-typescript-declarations.
+ */
```

Comment:
We should mention the source file that the user should edit, if they want to fix the generated typings.

Reply:
You mean the specific source files that it was derived from?

Reply:
Yeah, like this would mention paper-button-behavior.html (or .js or whatevs)

Reply:
Done! PTAL?